smpe/bld/service/current-hold-ACTION.txt

```diff
@@ -0,0 +1,34 @@
+****************************************************************
+* Affected function: Zowe configuration                        *
+****************************************************************
+* Description: redo configuration                              *
+****************************************************************
+* Timing: post-APPLY                                           *
+****************************************************************
+* Part: zowe-instance.env                                      *
+*       zowe-setup-certificate.env                             *
+****************************************************************
+New variables are added to zowe-instance.env and 
+zowe-setup-certificate.env.
+
+zowe-instance.env
+- ZOWE_EXPLORER_FRAME_ANCESTORS="${ZOWE_EXPLORER_HOST}:*,
+    ${ZOWE_IP_ADDRESS}:*"
+  # comma seperated list of hosts allowed to embed explorers
+
+Run <ROOT_DIR>/bin/zowe-configure-instance.sh -c 
+<PATH_TO_INSTANCE_DIR> again with an updated zowe-instance.env
+to pick up the new definitions.
```

Comment:
It doesn't have to be done on every upgrade, but might need to, so we don't count it in the 1-time only steps. Other examples include updating the XMEM and ZWESV proclibs (either manually, or using the the zowe-install-*.sh scripts). Again - in most (all since 1.9 so far) releases these don't change, but they are likely to at some point.